packages/react-magma-dom/src/i18n/default.ts

```diff
@@ -225,6 +225,9 @@ export const defaultI18n: I18nInterface = {
     nextButtonLabel: 'Scroll tabs forward',
     previousButtonLabel: 'Scroll tabs back',
   },
+  tag: {
+    deleteAriaLabel: 'Delete tag',
```

Comment:
This should be more descriptive. We do something in `Combobox` to use the `labelText`.

`'reset selection for {labelText}. {selectedItem} is selected',`. So here we could do `delete {labelText} tag`. Then in the `Tag`  component file you can do something like this ```const deleteAriaLabel = i18n.tag.deleteAriaLabel.replace(/\{labelText\}/g, labelText)```